packages/frontend/src/components/NavBar/HelpMenu/index.tsx

```diff
@@ -121,14 +121,34 @@ const HelpMenu: FC = () => {
                                 </IconContainer>
 
                                 <div>
-                                    <ItemCTA>Join the discussion</ItemCTA>
+                                    <ItemCTA>Join Slack community</ItemCTA>
                                     <ItemDescription>
                                         Get advice &amp; share best practices
                                         with other users.
                                     </ItemDescription>
                                 </div>
                             </HelpItem>
                         </LinkWrapper>
+                        <LinkWrapper
+                            role="button"
+                            href="Submit a feature request or bug report to improve Lightdash"
```

Comment:
This is not a valid URL , URL should be 

for bugs : https://github.com/lightdash/lightdash/issues/new?assignees=&labels=🐛+bug&template=bug_report.md&title= 

for features: https://github.com/lightdash/lightdash/issues/new?assignees=&labels=✨+feature-request&template=feature-request.md&title=

Other links might be available like https://github.com/lightdash/lightdash/issues/new/choose

For some reason, I can't see the changes on Render..

Reply:
haha dang. that was a copy paste error on my part! fixed :)